app/models/concerns/can_limit_number_of_people.rb

```diff
@@ -0,0 +1,81 @@
+module CanLimitNumberOfPeople
+  extend ActiveSupport::Concern
+
+  def enough_people?(type = person_type)
+    return false if number_of_existing_people(type) < minimum_people(type)
+    true
+  end
+
+  def can_only_have_one_person?(type = person_type)
```

Comment:
Just out of curiosity, why do we set `type` as `person_type` instead of just using `person_type` in the code?

Reply:
We want to be able to pass in a specific type if we want – this is mainly because CheckYourAnswers deals with multiple person_types so we need to validate them individually.